Lib/Src/CUartCom.cpp

```diff
@@ -98,25 +100,40 @@ void CUartCom::stopRx()
 }
 
 /**
- * @brief Add data to the TX Queue and start transmission if possible
- * @param msg Message to send.
- * @return True if message was added to the queue and transmission started
- * successfully.
+ * @brief send string
+ * @param msg Message to be sent via UART
+ * @return success if message was successfully stored in TX Queue.
  */
 bool CUartCom::send(etl::string<MAX_STRING_SIZE> msg)
 {
-    bool b_success = false;
-    // Add message to queue
-    if (m_tx_queue.size() <= MAX_TX_QUEUE_SIZE && (msg.empty() == false))
+    uint32_t msg_len = msg.length();
```

Comment:
This is very nice wrapper. Perhaps worth adding an `@note:` to the doxygen header to explain that this is a wrapper for pointer based `send()`.